### PR TITLE
Add RHINO Grafana Dashboard

### PR DESCRIPTION
**Pull request by ericjohncarlson**
## Description

Adding visibility to the RHINO Grafana/Prometheus dashboard on the Aptos community/external-resources page:
- Mix of chain, node and infrastructure metrics giving a strong understanding of the node operator's execution success
- Integration of [node-exporter](https://github.com/prometheus/node_exporter) and the integrated aptos-node prometheus endpoint
- Optional integration with [checklyhq.com](https://www.checklyhq.com/) for monitoring
- Optional integration with [Caddy](https://caddyserver.com/) for API use visibility

In use by multiple node ops today, so would like to create more visibility for other node operators

https://github.com/RhinoStake/aptos_monitoring

### CODE REVIEW

File: developer-docs-site/docs/community/external-resources.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ To add your own resource, click **Edit this page** at the bottom, add your resou
 
 | Contribution | Description | Author | Date added/updated |
 | --- | --- | --- | --- |
+| [Monitor Your Aptos Validator & VFN with Prometheus and Grafana](https://github.com/RhinoStake/aptos_monitoring) | A full featured Grafana/Prometheus dashboard to monitor key infrastructure, node, and chain related metrics and data relationships. | [RHINO](https://rhinostake.com) | 2023-03-10 |
```
Comment (clay-aptos) [inline developer-docs-site/docs/community/external-resources.md:18]: Here is a slight rewrite:

```
| [Monitor Your Aptos validator and validator fullnodes with Prometheus and Grafana](https://github.com/RhinoStake/aptos_monitoring) | A full-featured Grafana/Prometheus dashboard to monitor key infrastructure, node, and chain-related metrics and data relationships. | [RHINO](https://rhinostake.com) | 2023-03-10 |
```

**Review by clay-aptos — COMMENTED** [review]
Thanks, Eric! This LGTM with some small changes I suggest in a comment here. I am adding Michelle for review and asking her and our experts to also take a look before I approve. Thanks!

**Review by michelle-aptos — APPROVED** [review]

**Review by sherry-x — APPROVED** [review]

**Comment by ericjohncarlson** [comment]
Will do @clay-aptos.  Thanks!

**Comment by ericjohncarlson** [comment]
@clay-aptos merge conflict resolved.  Should be gtg.  Thanks again!

**Review by saharct — APPROVED** [review]
Thank you, @ericjohncarlson!

**Pull request merged**